esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -1006,6 +1067,64 @@ def create_timeseries_plot(self, datasets, short_name):
         with ProvenanceLogger(self.cfg) as provenance_logger:
             provenance_logger.log(plot_path, provenance_record)
 
+    def create_annual_cycle_plot(self, datasets, short_name):
+        """Create annual cycle plot."""
+        plot_type = 'annual_cycle'
+        if plot_type not in self.plots:
+            return
+
+        if not datasets:
+            raise ValueError(f"No input data to plot '{plot_type}' given")
+
+        logger.info("Plotting %s", plot_type)
+        fig = plt.figure(**self.cfg['figure_kwargs'])
+        axes = fig.add_subplot()
+
+        # Plot all datasets in one single figure
+        ancestors = []
+        for dataset in datasets:
+            ancestors.append(dataset['filename'])
+            cube = dataset['cube']
+            self._check_cube_dimensions(cube, plot_type)
+
+            # Plot annual cycle
+            plot_kwargs = self._get_plot_kwargs(plot_type, dataset)
+            plot_kwargs['axes'] = axes
+            iris.plot.plot(cube, **plot_kwargs)
+
+        # Default plot appearance
+        multi_dataset_facets = self._get_multi_dataset_facets(datasets)
+        axes.set_title(multi_dataset_facets['long_name'])
+        axes.set_xlabel('Month')
+        axes.set_ylabel(f"{short_name} [{multi_dataset_facets['units']}]")
+        axes.set_xticks(range(1, 13), [str(m) for m in range(1, 13)])
+
+        # Legend
+        legend_kwargs = self.plots[plot_type].get('legend_kwargs', {})
+        if legend_kwargs is not False:
+            axes.legend(**legend_kwargs)
+
+        # Customize plot appearance
+        self._process_pyplot_kwargs(plot_type, multi_dataset_facets)
+
+        # Save plot
+        plot_path = self.get_plot_path(plot_type, multi_dataset_facets)
+        fig.savefig(plot_path, **self.cfg['savefig_kwargs'])
+        logger.info("Wrote %s", plot_path)
+        plt.close()
+
+        # Provenance tracking
+        caption = (f"Annual cycle of {multi_dataset_facets['long_name']} for "
+                   f"various datasets.")
+        provenance_record = {
+            'ancestors': ancestors,
+            'authors': ['schlund_manuel'],
+            'caption': caption,
+            'plot_types': ['seas'],
```

Comment:
Thanks for reviewing @remi-kazeroni! `seas` refers to a 'seasonal cycle', which is a different expression for the annual cycle. I am using `annual_cycle` here since it is used in the other monitoring diagnostics.